packages/core/package.json

```diff
@@ -56,5 +56,12 @@
   "volta": {
     "extends": "../../package.json"
   },
+  "madge": {
+    "detectiveOptions": {
+      "ts": {
+        "skipTypeImports": true
+      }
+    }
+  },
```

Comment:
Circ deps (even types) are usually very smelly. Is this 100% necessary?

Reply:
yes, it is - as we need the util in `spanUtils`, and we need `spanUtils` in `span`, and for the util we need the span class from `span` 😬  at least I couldn't think of a way to make this work properly :( we can remove it in v8 though, then we don't need to import from `spanUtils` in `span` anymore!

Reply:
Alright then! Let's add a comment/todo somewhere then to remove this rule again (if you also think that it's worth keeping in general in the future).

Reply:
(or open for other ideas, of course)